test/async.js

```diff
@@ -38,20 +40,27 @@ test('reject an Error on missing `destination`', async t => {
 });
 
 test('copy a file', async t => {
-	await cpFile('license', t.context.destination);
+	await cpFile('license', t.context.destination, {clone: false});
 	t.is(fs.readFileSync(t.context.destination, 'utf8'), fs.readFileSync('license', 'utf8'));
 });
 
 test('copy an empty file', async t => {
 	fs.writeFileSync(t.context.source, '');
-	await cpFile(t.context.source, t.context.destination);
+	await cpFile(t.context.source, t.context.destination, {clone: false});
 	t.is(fs.readFileSync(t.context.destination, 'utf8'), '');
 });
 
 test('copy big files', async t => {
 	const buffer = crypto.randomBytes(THREE_HUNDRED_KILO);
 	fs.writeFileSync(t.context.source, buffer);
-	await cpFile(t.context.source, t.context.destination);
+	await cpFile(t.context.source, t.context.destination, {clone: false});
+	t.true(buffer.equals(fs.readFileSync(t.context.destination)));
+});
+
+test('clone a big file', async t => {
+	const buffer = crypto.randomBytes(THREE_HUNDRED_KILO);
+	fs.writeFileSync(t.context.source, buffer);
+	await cpFile(t.context.source, t.context.destination, {clone: true});
```

Comment:
Can't make this `force` yet -- I can't figure out how to use different install scripts for different OS. Since `cmd` is super incompatible with `sh` I don't think I can get away with some `if` statements. Otherwise I would be able to make a known-fs disk image and stuff.

Reply:
Oh, travis on Windows uses `mingw` sh. Guess I am free to do whatever.